internal/operands/template-validator/resources.go

```diff
@@ -42,6 +42,13 @@ const (
 
 func CommonLabels() map[string]string {
 	return map[string]string{
+		// Required Labels
+		common.AppKubernetesManagedByLabel: common.AppKubernetesManagedByValue,
+		common.AppKubernetesVersionLabel:   common.GetOperatorVersion(),
+		common.AppKubernetesComponentLabel: VirtTemplateValidator,
+		common.AppKubernetesPartOfLabel:    common.AppKubernetesPartOfValue,
```

Comment:
These labels are already added, see:
https://github.com/kubevirt/ssp-operator/blob/e92bc396eca2195d7d43b30d8f9fb363fb418f47/internal/common/labels.go#L36-L49

Reply:
Yes, but this is no used by `virt-template-validator`, specifically `WithAppLabels()`. There is just `CommonLabels()`. Should we use `WithAppLabels()`` instead of `CommonLabels()`?